source/community/reactnative/android/app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,7 @@
       android:label="@string/app_name"
       android:icon="@mipmap/ic_launcher"
       android:allowBackup="false"
+	  android:usesCleartextTraffic="true"
```

Comment:
Though this fixes the issue for visualizer app (which is a test app).
What impact will it have on 3rd-party host apps which strictly require this attribute to be 'false'?

Reply:
@Zaidos9 This will be required only during dev time, to allow the app to connect to the bundler. It wont be necessary in the production versions as the JS modules will be bundled within the APK. So its host application's responsibility to have this configured for release and dev modes.

Reply:
@Zaidos9 I am marking this as resolved. Please reopen if required.